### ISSUE TITLE
Enabling plugin breaks modal

### ISSUE THREAD

**Issue by PhilippSPOTTERON** [mostafa272/Gif-Player#3]
Hi,
another issue is that when the plugin is enabled, a modal box built in our website at www.spotteron.net doesnt work anymore:

(index):133 Uncaught TypeError: a(...).jf_login_modal is not a function
    at (index):133
    at dispatch (jquery-1.11.2.min.js:3)
    at r.handle (jquery-1.11.2.min.js:3)

The modal is the black login icon on top right.
With an unpublished plugin it works normally.

**Comment by mostafa272** [comment]
@PhilippSPOTTERON  Hi, It should be a conflict in jQuery codes. Open gifplayer.php and in lines 19 and 20 change dollar signs ($) to jQuery.
If replacing dollar signs with jQuery was not useful, try to remove line 17 of gifplayer.php. This line calls jquery-1.11.2.min.js. I hope it works for you.

**Comment by PhilippSPOTTERON** [comment]
Hi, thanks for the reply,
i changed one of the two in those lines like that:
$script="$(document).ready( function(){
				jQuery('.gifplayer').gifplayer({label:'".$label."'});

			});";

but I am not sure what to do with the .$label and $script. - I tried jQuery(label) but that just makes a 500 internal server error. or is it jQueryscript and jQuerylabel?

**Comment by mostafa272** [comment]
@PhilippSPOTTERON No, read my previous comment again. Just change $(document) to jQuery(document) and in next line change $('.gifplayer') to jQuery('.gifplayer'). $label is a PHP variable and this problem is related to Javascript. Don't change it.

**Comment by PhilippSPOTTERON** [comment]
Thanks for your clearification,
I am coming from a design background and while good at css and art tools, I am not so experienced in programming from scratch.
I changed it according to your instructions but it still breaks the modal login box at www.spotteron.net

code:
$script="jQuery(document).ready( function(){
				jQuery('.gifplayer').gifplayer({label:'".$label."'});

**Comment by mostafa272** [comment]
@PhilippSPOTTERON Do you try to remove line 17 ? code of this line is this:                       


`$document->addScript('https://code.jquery.com/jquery-1.11.2.min.js');`

**Comment by PhilippSPOTTERON** [comment]
Ok, super, that did the trick!
Works again now, it seems this created the conflict.
Thanks.

**Comment by mostafa272** [comment]
@PhilippSPOTTERON  not at all, Philip.
